graphene_django/utils/utils.py

```diff
@@ -26,7 +26,7 @@ def isiterable(value):
 
 def _camelize_django_str(s):
     if isinstance(s, Promise):
-        s = force_text(s)
+        s = force_str(s)
```

Comment:
Is it okay with Django 2.2?

Reply:
I'm not sure how I would test that version specifically, but all the project tests pass in my fork https://github.com/MisterGlass/graphene-django/pull/2

Reply:
Ah, wait, here is the 2.2 test pass https://github.com/MisterGlass/graphene-django/runs/4385256293?check_suite_focus=true